backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -64,4 +69,13 @@ public void updateUserById(
 		userService.updateUser(user.getId(), userUpdateModel.getEmail());
 	}	
 
+	@PostMapping
+	@Operation(summary = "Create a new user ")
+	public void createUserByEmailAndUserAttributes(
+
+		@Validated @RequestBody UserCreateModel user) {
+			List<UserAttributeModel> userAttributesModels = user.getUserAttributeModel();
+
+			userService.createUser(user.getEmail(), userAttributesModels);
```

Comment:
You should use a mapper method to convert from the model to the domain object, then call `userService.createUser(..)` with that domain object.